modules/UI/videolayout/VideoContainer.js

```diff
@@ -495,18 +527,15 @@ export class VideoContainer extends LargeContainer {
         }
 
         stream.attach(this.$video[0]);
-        stream.attach(this.$videoBackground[0]);
-
-        this._hideVideoBackground();
 
         const flipX = stream.isLocal() && this.localFlipX;
 
         this.$video.css({
             transform: flipX ? 'scaleX(-1)' : 'none'
         });
-        this.$videoBackground.css({
-            transform: flipX ? 'scaleX(-1)' : 'none'
-        });
+
+        this._hideBackground = true;
```

Comment:
why does this block unconditionally hide the background?

Reply:
I think it hides it because the stream has just changed and it's waiting for the stream to load. Something else will react to the video loading and call resize, which would then show the background. Would you prefer if I tried removing the hiding and see what the effect might be?

Reply:
I've removed it. I can't remember why I left it in. I did a quick test without it in and things...seemed fine. I guess we'll see if it breaks anything...